provider/testdata/data/json/displaynames/languages@1/en.json

```diff
@@ -325,6 +327,7 @@
     "kw": "Cornish",
     "kwk": "Kwakʼwala",
     "ky": "Kyrgyz",
+    "ky-alt-variant": "Kirghiz",
```

Comment:
Suggestion: We should drop the alt-variant; there's another issue, https://github.com/unicode-org/icu4x/issues/1225, to solve this across the board.

Reply:
Done.